.gitpod/Dockerfile

```diff
@@ -0,0 +1,27 @@
+#Pull Gitpod Workspace Image
+FROM gitpod/workspace-full
+
+#Pull Postgres Image
+FROM gitpod/workspace-postgres
```

Comment:
You are pulling the first image but aren't doing anything with it. 
Do you mean to base your image from the first or second one?

Reply:
I am extending the Gitpod base image `gitpod/workspace-full`because it provides many tools which might be required including Docker support which I am quoting below from the documentation

`Docker support: If you use the gitpod/workspace-full image, you get Docker support built-in to your environment.`

I hope this clarifies my intention

Reply:
In this case, you need to remove the line

```
FROM gitpod/workspace-postgres
```

Reply:
Please read https://docs.docker.com/engine/reference/builder/#from and other resources about Docker and Dockerfile because here I'm not sure you see exactly what is happening with your instructions.

Reply:
Thank you @alecharp for providing the documentation. After some research, I found that one should not put two base images in a single Dockerfile.  Docker will use only the `gitpod/workspace-full` base image and ignore the `gitpod/workspace-postgres` base image.

I have removed the `gitpod/workspace-full` base image as it is heavy and includes many tools and language support like `python` which might not be useful. 

`gitpod/workspace-postgres` contains support for Docker and has postgreSQL installed along with many utility tools for it. We are subsequently setting the environment for Java, Node.js and Maven. Yarn comes pre-installed with the image.

I have tested it by running on Gitpod.

Reply:
Hi @alecharp, does my latest code change fix what you were hinting at?